### PR TITLE
changes for backup, restore, restart

### PR DESCRIPTION
**Pull request by jsingle**
@paetling

### CODE REVIEW

**Comment by jsingle** [comment]
You know I have other documentation changes I should make as well

File: docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## 0.1.3 (In Progress)
+  * NEW: `dusty restart --repos <repos>` will restart active containers which use the specified repos.
+  * NEW: `dusty disk backup <destination>` will save the contents of your VM's `/persist` directory
+  * NEW: `dusty disk restore <source>` will write to your VM's `/persist` from `<source>`
```
Comment (paetling) [inline docs/changelog.md]: I like the pattern that NEW: 'dusty status' had.  It stated what has been added and what this new command does. Makes it a little more clear that this is a new command.  Not sold one way or the other, thoughts?

Comment (jsingle) [inline docs/changelog.md]: Yeah I think it's a little clearer like that

**Comment by paetling** [comment]
one ocmment but looks good. :ghost: :cow:

**Pull request merged**